src/pudl/glue/ferc1_eia.py

```diff
@@ -124,26 +127,36 @@ def get_mapped_plants_eia():
 ##########################
 
 
-def get_util_ids_ferc1_raw_xbrl(ferc1_xbrl_engine: sa.engine.Engine) -> pd.DataFrame:
-    """Grab the utlity ids (reported as `entity_id`) in the FERC1 XBRL database."""
+def string_normalize_col_ferc1(df: pd.DataFrame, col) -> pd.DataFrame:
```

Comment:
I feel like we just did a bunch of work to avoid having these kinds of functions proliferating... can we just the existing `normalize_strings_multicol()` here with a dictionary containing a single element instead if you want to put it in a method chain?

```py
from pudl.transform.classes import normalize_strings_multicol, StringNormalization
from pudl.transform.params.ferc1 import FERC1_STRING_NORM
params = {"utility_name_ferc1": StringNormalization(**FERC1_STRING_NORM)}
df = normalize_strings_multicol(df, params)
```

Or it could be done with an assign:
```py
df = df.assign(utility_name_ferc1=lambda x: normalize_strings(x.utility_name_ferc1, params))
```